src/main/java/binnie/extrabees/genetics/ExtraBeeDefinition.java

```diff
@@ -1649,7 +1649,7 @@ protected void setSpeciesProperties(IAlleleBeeSpeciesCustom beeSpecies) {
 
 		@Override
 		protected void registerMutations() {
-			registerMutation(BeeDefinition.ENDED, RELIC, 100)
+			registerMutation(BeeDefinition.ENDED, RELIC, 2)
```

Comment:
Because chance of this mutation is 2 in original Binnie: https://github.com/ForestryMC/Binnie/blob/master-MC1.7.10/src/main/java/binnie/extrabees/genetics/ExtraBeeMutation.java#L203
I did 100% for the test condition and accidentally made a commit with this.